cmd/endpoint-ellipses.go

```diff
@@ -129,6 +112,44 @@ func getSetIndexes(args []string, totalSizes []uint64, customSetDriveCount uint6
 			return nil, config.ErrInvalidErasureSetSize(nil).Msg(msg)
 		}
 		setSize = customSetDriveCount
+		globalCustomErasureDriveCount = true
+	} else {
+		var newSetCounts = make(map[uint64]struct{})
+		for _, ss := range setCounts {
+			var valid bool
+			for _, argPattern := range argPatterns {
+				for _, p := range argPattern {
+					if uint64(len(p.Seq)) > ss {
+						valid = uint64(len(p.Seq))%ss == 0
+					} else {
+						valid = ss%uint64(len(p.Seq)) == 0
```

Comment:
Potential division by 0.

Do we have any checks on whether `len(p.Seq)` can be 0?

Reply:
Yes it won't ever be passed down, you cannot provide 0...0 ellipses.